### DISCUSSION TITLE
when [ Press Ctrl + Enter],  new page show only one accept solution for user selection

### DISCUSSION THREAD

**Discussion by yifanff** [community/community#10718] [Copilot Conversations] [answered]
As for function " Getting more suggestions", when Press Ctrl+Enter, GitHub Copilot will open a new tab and should suggest multiple options, but **only one accept solution in the new tab**, although it said "Synthesizing 8/10 solutions "

![image](https://user-images.githubusercontent.com/5958693/151652587-55af996d-0d31-4c5e-9d16-9e9b7fee0dd0.png)

**Comment by AcmeGamers** [comment, marked as answer]
You can try re-opening the VS Code, it should work properly. In case it doesn't, try reinstalling the extension. I hope this helps you out

**Reply by yifanff** [reply]
OK, Thanks, AcmeGamers.
